cdc/sink/codec/interface.go

```diff
@@ -63,9 +63,13 @@ type MQMessage struct {
 	Protocol Protocol            // protocol
 }
 
+// producerMessageOverhead is used to calculate ProducerMessage's byteSize by sarama kafka client.
+// reference: https://github.com/Shopify/sarama/blob/66521126c71c522c15a36663ae9cddc2b024c799/async_producer.go#L231
+const producerMessageOverhead = 26 // the metadata overhead of CRC, flags, etc.
```

Comment:
Is this constant related to Kafka version? when `version >= 2`, maximumRecordOverhead is 36
https://github.com/Shopify/sarama/blob/66521126c71c522c15a36663ae9cddc2b024c799/async_producer.go#L235-L242

Reply:
yes, we use `2.4.0` as default, is it possible to be `version < 2`?

Reply:
it must be version `2`.